geode-core/src/main/java/org/apache/geode/internal/cache/wan/InternalGatewaySender.java

```diff
@@ -40,6 +40,8 @@
 
   boolean isForwardExpirationDestroy();
 
+  boolean isPartitionedRegionClearUnsupported();
```

Comment:
I think it would be clearer to use {{isPartitionedRegionClearSupported()}} instead of the negative "unsupported".

Reply:
@kirklund That would mean changing the default to true. Does that sound good to you?

Reply:
We just changed to "unsupported" based on Darrel's comments.